source/common/http/conn_manager_impl.h

```diff
@@ -787,7 +784,6 @@ class ConnectionManagerImpl : Logger::Loggable<Logger::Id::http>,
   const Server::OverloadActionState& overload_stop_accepting_requests_ref_;
   const Server::OverloadActionState& overload_disable_keepalive_ref_;
   TimeSource& time_source_;
-  std::shared_ptr<StreamInfo::FilterState> filter_state_;
```

Comment:
This is the filter state that was (lazily) created by a network filter. I changed it to use the filter state from the connection stream info.